boot/bootutil/src/image_ec256.c

```diff
@@ -127,17 +129,19 @@ tinycrypt_decode_sig(uint8_t signature[NUM_ECC_BYTES * 2], uint8_t *cp, uint8_t
         return -2;
     }
 
-    rc = tinycrypt_read_bigint(signature, &cp, end);
+    rc = bootutil_read_bigint(signature, &cp, end);
     if (rc) {
         return -3;
     }
-    rc = tinycrypt_read_bigint(signature + NUM_ECC_BYTES, &cp, end);
+	
+	rc = bootutil_read_bigint(signature + NUM_ECC_BYTES, &cp, end);
```

Comment:
weird ident

Reply:
I'll fix it seems like an editor mistake tnx for noticing.